octobot_evaluators/evaluators/strategy_evaluator.py

```diff
@@ -115,6 +117,9 @@ async def stop(self) -> None:
             await evaluator_channels.get_chan(constants.MATRIX_CHANNEL,
                                               self.matrix_id).remove_consumer(self.consumer_instance)
 
+    def get_full_cycle_evaluator_types(self) -> tuple:
```

Comment:
Why a tuple and not a list?

Reply:
A tupe is faster than a list to create and we don't need to edit later on it I think it's better than a list in this case

Reply:
Ok :+1: Can you add a comment to prevent later refactor please?